panel/template/base.py

```diff
@@ -449,6 +449,9 @@ class BasicTemplate(BaseTemplate, ResourceComponent):
     header = param.ClassSelector(class_=ListLike, constant=True, doc="""
         A list-like container which populates the header bar.""")
 
+    initial_sidebar_state = param.Selector(default='expanded', objects=['expanded', 'collapsed'], doc="""
```

Comment:
Do you expect that there will be more than two states? If not, couldn't this just be a boolean? If a boolean, then ignore my notebook suggestions.

Reply:
I was copying the `streamlit` API for this, do agree though.